specs/deneb/p2p-interface.md

```diff
@@ -309,19 +311,22 @@ Clients SHOULD include a sidecar in the response as soon as it passes the gossip
 Clients SHOULD NOT respond with sidecars related to blocks that fail gossip validation rules.
 Clients SHOULD NOT respond with sidecars related to blocks that fail the beacon chain state transition
 
-##### BlobSidecarsByRange v1
+For each `response_chunk`, a `ForkDigest`-context based on `compute_fork_version(compute_epoch_at_slot(blob_sidecar.signed_block_header.message.slot))` is used to select the fork namespace of the Response type.
 
-**Protocol ID:** `/eth2/beacon_chain/req/blob_sidecars_by_range/1/`
+Per `context = compute_fork_digest(fork_version, genesis_validators_root)`:
 
-*[New in Deneb:EIP4844]*
+[0]: # (eth2spec: skip)
 
-The `<context-bytes>` field is calculated as `context = compute_fork_digest(fork_version, genesis_validators_root)`:
+| `fork_version`                 | Chunk SSZ type                |
+|--------------------------------|-------------------------------|
+| `DENEB_FORK_VERSION` and later | `deneb.BlobSidecar`           |
```

Comment:
"and later"? I saw that we didn't use this phase at all (except in light-client spec).